x-pack/plugins/uptime/public/components/queries/error_list/error_list.tsx

```diff
@@ -22,38 +24,74 @@ export const ErrorList = ({ dateRangeStart, dateRangeEnd, filters }: ErrorListPr
   <Query query={getErrorListQuery} variables={{ dateRangeStart, dateRangeEnd, filters }}>
     {({ loading, error, data }) => {
       if (loading) {
-        return 'Loading...';
+        return i18n.translate('xpack.uptime.errorList.loadingMessage', {
+          defaultMessage: 'Loading...',
+        });
       }
       if (error) {
-        return `Error ${error.message}`;
+        return i18n.translate('xpack.uptime.errorList.errorMessage', {
+          values: { message: error.message },
+          defaultMessage: 'Error {message}',
+        });
       }
       const { errorList } = data;
       return (
         <Fragment>
           <EuiTitle size="xs">
-            <h5>Error list</h5>
+            <h5>
+              <FormattedMessage id="xpack.uptime.errorList.title" defaultMessage="Error list" />
+            </h5>
           </EuiTitle>
           <EuiPanel>
             <EuiInMemoryTable
               items={errorList}
               columns={[
-                { field: 'type', name: 'Error Type', sortable: true },
+                {
+                  field: 'type',
+                  name: i18n.translate('xpack.uptime.errorList.columns.type', {
```

Comment:
```suggestion
                  name: i18n.translate('xpack.uptime.errorList.errorTypeColumnLabel', {
```

please apply for the similar cases the pattern: `xpack.uptime.errorList.[columnName]ColumnLabel'`